wcfsetup/install/files/style/layout/global.scss

```diff
@@ -197,6 +197,12 @@ ol.nativeList {
 			margin: 0 0 20px 20px;
 		}
 	}
+
+	.messageFloatObjectLeft, .image-style-side-left, .messageFloatObjectRight, .image-style-side {
+		+ p, + ul, + ol, + h1, + h2, + h3, + h4, + h5, + h6 {
```

Comment:
This creates a lot of selectors, you should be able to simplify this with [`:is()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is).

Reply:
Also, this selector is wrong, because it unconditionally targets the floating objects regardless of their position within the flow. You may only strip the top margin if this is actually the first direct child of the message text.